packages/sw-lib/src/index.js

```diff
@@ -26,6 +26,54 @@ if (!assert.isSWEnv()) {
   throw ErrorFactory.createError('not-in-sw');
 }
 
+/**
+ * The sw-lib module is a high-level library that makes it easier to
+ * configure routes with caching strategies as well as manage precaching
+ * of assets during the install step of a service worker.
+ *
+ * @example
+ * importScripts('/<Path to Module>/build/sw-lib.min.js');
+ *
+ * // cacheRevisionedAssets() can take an array of strings with
+ * // the revision details in the url.
+ * goog.swlib.cacheRevisionedAssets([
+ *     '/styles/main.1234.css',
+ *     '/images/logo.abcd.jpg'
+ * ]);
+ *
+ * // Or it can accept objects with the URL and revision data
+ * // kept seperate.
+ * goog.swlib.cacheRevisionedAssets([
+ *     {
+ *       url: '/index.html',
+ *       revision: '1234'
+ *     },
+ *     {
+ *       url: '/about.html',
+ *       revision: 'abcd'
+ *     }
+ * ]);
+ *
+ * // If you have assets that aren't revisioned, you can cache them
```

Comment:
At the moment the precaching module doesn't configure fetch routes, so technically both require "run time caching strategies".